phpunit.xml.dist

```diff
@@ -18,6 +19,11 @@
     <testsuites>
         <testsuite name="MongoDB CRUD Test Suite">
             <directory>./tests/</directory>
+            <exclude>tests/SpecTests/AtlasDataLakeSpecTest.php</exclude>
```

Comment:
I suppose this is optional because `AtlasDataLakeSpecTest.php` should already skip if the server doesn't identify as ADL. Up to you if you'd prefer to keep it, but I think removing this would ensure that the skip logic actually gets tested.

Reply:
That is correct, we could remove this and rely on the skip logic. This has also become unnecessary with the introduction of the `defaultTestSuite` option. Before that, invoking PHPUnit without a `--testsuite` argument would run all test suites, so the ADL tests would get run twice if it wasn't excluded here.

I'm removing the exclude since it's not necessary anymore.